Unicode operators ≤≥≠≢≡≣⩶⩵«»⋙‖⁇∈∉∋∌▷‥…≔→⇒

edemaine opened the pull request:
For fun, here are some simpler and less controversial Unicode operators from #555.

The only one I'm less sure about is `≡` mapping to `===`. But given that `≠` makes `!=` and `≢` makes `!==`, this feels natural. And anyway it's good to encourage `===` over `==`. Alternatively, we could use `≣` for `===` and `≡` for `==`...

Incidentally, TIL there are symbols for double and triple equals: `⩵` and `⩶`.

Going forward:
* I'm less sure about the symbols with equals and such because they look like comparisons to me, but Civet uses them as assignments.
* I think the power operators, specifically `²`, would be handy, but that's a little more complicated, so left for the future.
* If there are other Unicode operators you think I should add now, let me know!

STRd6 commented:
I think these are good. `≡` mapping to `===` makes sense to me. `∈` and `∉` are cool. Looking forward to exploring more in the future!

GoToLoop commented:
> , we could use `≣` for `===` and `≡` for `==`...

I prefer this combo, I guess. 💭

edemaine commented:
> we could use `≣` for `===` and `≡` for `==`...

This would follow the pattern: for every line beyond `=`, you add another `=`. @STRd6 what do you think? (With `coffeeEq`, I would imagine `≡` still maps to `===`, just like `==` does.)

STRd6 commented:
> > we could use `≣` for `===` and `≡` for `==`...
> 
> This would follow the pattern: for every line beyond `=`, you add another `=`. @STRd6 what do you think? (With `coffeeEq`, I would imagine `≡` still maps to `===`, just like `==` does.)

This sounds good, either way is fine with me.

edemaine commented:
OK, I switched, and also added → and ⇒ for functions (obvious in hindsight!) and inclusion operators ∋ and ∌. Plus tests and docs. Let me know if there are any final comments before merging.

STRd6 commented:
Looks good to me! 👍
